Sources/ReleaseServer/ReleaseServer.WebApi/Mappers/DeploymentMetaMapper.cs

```diff
@@ -1,14 +1,20 @@
 using System.IO;
+using System.Text;
 using Newtonsoft.Json;
 using ReleaseServer.WebApi.Config;
 
 namespace ReleaseServer.WebApi.Mappers
 {
     public static class DeploymentMetaInfoMapper
     {
-        public static DeploymentMetaInfo ParseDeploymentMetaInfo(string fileName)
+        public static DeploymentMetaInfoModel ParseDeploymentMetaInfo(string fileName)
         {
-            return JsonConvert.DeserializeObject<DeploymentMetaInfo>(File.ReadAllText(fileName));
+            return JsonConvert.DeserializeObject<DeploymentMetaInfoModel>(File.ReadAllText(fileName));
+        }
+        
+        public static DeploymentMetaInfoModel ParseDeploymentMetaInfo(byte[] byteContent)
+        {
+            return JsonConvert.DeserializeObject<DeploymentMetaInfoModel>(Encoding.UTF8.GetString(byteContent));
```

Comment:
This method is only used in the test project. So it seems to be obsolete.

Reply:
You are right, It got removed in the second PR:)